src/components/Buttons/LoadingButton.tsx

```diff
@@ -13,7 +13,8 @@ export const LoadingButton: ExtendableBox<
 > = forwardRef(({ leftGlyph, loading = false, ...rest }: Props, ref) => (
```

Comment:
The type definition above could be tightened up a little bit, I think; it currently includes `loading` and `isLoading`. We should probably choose one or the other for clarity.

Reply:
oops thanks sophie! I kind of like `loading` more so I kept that one